eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -0,0 +1,472 @@
+. (Join-Path $PSScriptRoot SemVer.ps1)
+
+$SDIST_PACKAGE_REGEX = "^(?<package>.*)\-(?<versionstring>$([AzureEngSemanticVersion]::SEMVER_REGEX.Substring(1)))"
```

Comment:
The substring(1) is a little odd and might break in the future. It might be better to remove the `^` and `$` from the REGEX especially if we plan to use it in ways like this.

Reply:
I'll push one additional commit.